File: Fizzbuzz.java
import java.util.*;

public class Fizzbuzz {

	public static void main(String[] args) {
		for(int i=0; i<101; i++) {
			System.out.print(i + " ");
			if(i % 3 == 0)
				System.out.print("Fizz");
			if(i % 5 == 0)
				System.out.print("Buzz");

			System.out.print("\n");
		}
	}
}

File: fizzbuzz.php
<!doctype html>
<html>
<head>
	<title>FizzBuzz in php!</title>
</head>
<body>
<?php
	$count = 0;
	while ( $count <= 100 ) {
		echo "$count ";
		if($count % 3 == 0) {
			echo "Fizz";
		}
		if($count % 5 ==0) {
			echo "Buzz";
		}
		echo "<br />";
		$count++;
	}
?>
</body>
</html>
